README.md

```diff
@@ -5,3 +5,17 @@ Shared code and documentation for Becoming a teacher infrastructure.
 * [Monitoring](monitoring/readme.md)
 * [CI/CD strategy](documentation/ci-cd-strategy.md)
 * [GitHub Actions](actions/README.md)
+
+## Check paas users
+Only authorised users may have access to production spaces. The powershell script `check-users-in-space-developer-role.ps1` lists the users and
+either report to Slack or automatically delete the users. A whitelist must be provided so required users like service accounts are not deleted.
+
+List extra users:
+```
+% ./scripts/check-users-in-space-developer-role.ps1 <space_name> <slack_webhook_url>
+```
+
+Delete extra users:
+```
+% ./scripts/check-users-in-space-developer-role.ps1 <space_name> <slack_webhook_url> delete <protected_user_names_comma_separated_list>
+```
```

Comment:
i think it is worth adding that to get the list of parameters, you can do:-

`check-users-in-space-developer-role.ps1 - ` + tab will give the list of all input parameters too.

Reply:
Tested and this only works in a powershell shell. The feature is of limited use